data/mods/Aftershock/maps/afs_item_groups.json

```diff
@@ -237,42 +269,7 @@
       [ "afs_textbook_shotguns", 3 ],
       [ "afs_textbook_handguns", 3 ],
       [ "afs_textbook_rifles", 3 ],
-      [ "afs_textbook_launchers", 3 ],
-      [ "textbook_atomic", 8 ]
-    ]
-  },
-  {
-    "id": "guns_pistol_improvised",
-    "type": "item_group",
-    "items": [ [ "v29_cheap", 10 ] ]
```

Comment:
The homemade laser pistol is now impossible to acquire.

Reply:
Huh, was there no recipe for it in BN? I'm sorry, I must have mised that. I'll put something in real quick for it.